packages/console/src/pages/SignInExperience/tabs/SignUpAndSignIn/SignUpForm.tsx

```diff
@@ -80,7 +81,11 @@ const SignUpForm = () => {
         {
           identifier: requiredIdentifier,
           password: getSignInMethodPasswordCheckState(requiredIdentifier, isSignUpPasswordRequired),
-          verificationCode: getSignInMethodVerificationCodeCheckState(requiredIdentifier),
+          verificationCode: getSignInMethodVerificationCodeCheckState(
```

Comment:
According to the issue description, should not update the verification code value in `signInMethods`?   Why do we need to evaluate the `verificationCode` here?